src/patternfly/components/Page/examples/Page.md

```diff
@@ -123,8 +123,8 @@ cssPrefix: pf-c-page
     {{#> page-main-section page-main-section--modifier="pf-m-no-padding pf-m-light"}}
       This `.pf-c-page__main-section` uses `.pf-m-no-padding` to remove all padding.
     {{/page-main-section}}
-    {{#> page-main-section page-main-section--modifier="pf-m-no-padding-mobile"}}
-      This `.pf-c-page__main-section` uses `.pf-m-no-padding-mobile` to remove padding on mobile only.
+    {{#> page-main-section page-main-section--modifier="pf-m-no-padding-on-md"}}
```

Comment:
I think this is going to need to be:

```suggestion
    {{#> page-main-section page-main-section--modifier="pf-m-no-padding pf-m-padding-on-md"}}
```

I wonder if we should keep `no-padding-mobile` for places that already have it like this, and support removing/adding padding at the breakpoints? And `no-padding-mobile` can always represent removing the padding at whatever breakpoint we change the page to a mobile view.

Reply:
per offline conversation on slack, we decided to remove `no-padding-mobile` and use `pf-n-no-padding pf-m-padding-on-md` and then this shouldnt have any change to how its used in React.